### DISCUSSION TITLE
Enjin userdata base scrape

### DISCUSSION THREAD

**Discussion by QPfixxy** [allquixotic/encuum#8] [Q&A]
What is the likeliness of the feature related to user data being implemented? Currently, our community has approximately 28k members on it. My knowledge of web scraping is minor, so this is a monumental task (at least for me).

**Comment by allquixotic** [comment]
Hi, can you specify what specific data you're looking for? I just want to be clear on what exactly you need most urgently. I'm currently working on website applications (the feature where someone can fill out a form to apply to your Enjin website). I should have time to get to some additional features, but I won't be working on anything Minecraft-specific.

**Reply by QPfixxy** [reply]
Specifically I’m trying to figure out how I could collect the usernames/tags of every user we have registered to our enjin site.

**Reply by QPfixxy** [reply]
It would also be super useful to collect their assigned website tags, as those correspond to donations/ranks they’ve acquired.

**Comment by allquixotic** [comment]
Thanks for the feedback! I definitely plan to collect the information you requested. It's just a matter of finding time to implement the code. I'm planning to have this stuff done in February, so stay tuned!
